src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { props, skinnable, t } from 'revenge';
+import './search-bar.scss';
+
+@skinnable()
+@props({
+  value: t.String,
+  onChange: t.Function
+})
+export default class SearchBar extends React.Component {
+
+  onSearch = () => {
+    const input = React.findDOMNode(this.refs.input);
+    this.props.onChange(input.value);
```

Comment:
you shouldn't access a form field directly. instead, use their API: `value` and `onChange`.
in this case, as `value` and `onChange` are passed as `props`, you could simply do:
```
<input value={locals.value} onChange={locals.onChange} {...} />
```